functorch/dim/__init__.py

```diff
@@ -2,6 +2,7 @@
 from typing import Union, Sequence
 import inspect
 import dis
+from .einops import rearrange as rearrange
```

Comment:
Do you want `einops` or `rearrange` imported here so that ppl can do

```python
import functorch

functorch.rearrange
# or
functorch.einops.rearrange
```

?